src/ONYXKEYS.ts

```diff
@@ -291,6 +291,7 @@ const ONYXKEYS = {
         PRIVATE_NOTES_FORM: 'privateNotesForm',
         I_KNOW_A_TEACHER_FORM: 'iKnowTeacherForm',
         INTRO_SCHOOL_PRINCIPAL_FORM: 'introSchoolPrincipalForm',
+        REPORT_FRAUD_FORM: 'reportFraudForm',
```

Comment:
Maybe also could make this: `REPORT_VIRTUAL_CARD_FRAUD`? Not sure if we will have other kinds of fraud reporting, but doesn't hurt to get specific here.

Reply:
Updated!